tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -174,4 +178,308 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
+	})
+
+	/*
+		Verify large EZT volume creation (should take >vpxd task timeout)
+		Steps:
+		1. Create a SPBM policy with EZT volume allocation
+		2. Create a SC using policy created in step 1
+		3. Create a large PVC using SC created in step 2, this should take more than vpxd task timeout
+		4. Wait and verify the PVC created in step 3 is bound
+		5. Verify no orphan volumes are created
+		6. Delete the PVC created in step 3
+		7. Delete the SC created in step 2
+		8. Deleted the SPBM policy created in step 1
+	*/
+	ginkgo.It("[csi-block-vanilla] Verify large EZT volume creation which takes longer than vpxd timeout", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		sharedvmfsURL := os.Getenv(envSharedVMFSDatastoreURL)
+		if sharedvmfsURL == "" {
+			ginkgo.Skip(fmt.Sprintf("Env %v is missing", envSharedVMFSDatastoreURL))
+		}
+
+		largeSize := os.Getenv(envDiskSizeLarge)
+		if largeSize == "" {
+			largeSize = diskSizeLarge
+		}
+
+		scParameters := make(map[string]string)
+
+		rand.Seed(time.Now().UnixNano())
+		suffix := fmt.Sprintf("-%v-%v", time.Now().UnixNano(), rand.Intn(10000))
+		categoryName := "category" + suffix
+		tagName := "tag" + suffix
+
+		catID, tagID := createCategoryNTag(ctx, categoryName, tagName)
+		defer func() {
+			deleteCategoryNTag(ctx, catID, tagID)
+		}()
+
+		attachTagToDS(ctx, tagID, sharedvmfsURL)
+		defer func() {
+			detachTagFromDS(ctx, tagID, sharedvmfsURL)
+		}()
+
+		ginkgo.By("Create a SPBM policy with EZT volume allocation")
+		policyID, policyName := createVmfsStoragePolicy(
+			ctx, pc, eztAllocType, map[string]string{categoryName: tagName})
+		defer func() {
+			deleteStoragePolicy(ctx, pc, policyID)
+		}()
+
+		setVpxdTaskTimeout(vpxdReducedTaskTimeoutSecsInt)
+		defer func() {
+			setVpxdTaskTimeout(0)
+		}()
+
+		ginkgo.By("Create SC using policy created in step 1")
+		ginkgo.By("Create a large PVC using SC created in step 2, this should take more than vpxd task timeout")
+		scParameters[scParamStoragePolicyName] = policyName
+		storageclass, pvclaim, err := createPVCAndStorageClass(client,
+			namespace, nil, scParameters, largeSize, nil, "", false, "")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		defer func() {
+			ginkgo.By("Delete the SC created in step 2")
+			err := client.StorageV1().StorageClasses().Delete(ctx, storageclass.Name, *metav1.NewDeleteOptions(0))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}()
+
+		start := time.Now()
+		ginkgo.By("Verify the PVCs created in step 3 are bound")
+		pvs, err := fpv.WaitForPVClaimBoundPhase(
+			client, []*v1.PersistentVolumeClaim{pvclaim}, framework.ClaimProvisionTimeout)
+		elapsed := time.Since(start)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Verify PVC creation took longer than vpxd timeout")
+		gomega.Expect(elapsed > time.Second*time.Duration(vpxdReducedTaskTimeoutSecsInt)).To(
+			gomega.BeTrue(), "PVC creation was faster than vpxd timeout")
+
```

Comment:
Please add a log stating that PVC creation took longer than vpxd timeout.

Reply:
not sure if it adds any more value since we have
ginkgo.By("Verify PVC creation took longer than vpxd timeout")

and then continue on to next steps